src/lib/key_value_database/key_value_database.ml

```diff
@@ -0,0 +1,61 @@
+open Core_kernel
+
+module type S = sig
+  type t
+
+  type key
+
+  type value
+
+  val create : directory:string -> t
+
+  val close : t -> unit
+
+  val get : t -> key:key -> value option
+
+  val set : t -> key:key -> data:value -> unit
+
+  val remove : t -> key:key -> unit
+end
+
+module type Mock_intf = sig
+  include S
+
+  val random_key : t -> key option
+
+  val to_sexp :
+    t -> key_sexp:(key -> Sexp.t) -> value_sexp:(value -> Sexp.t) -> Sexp.t
+end
+
+module Make_mock
+    (Key : Hashable.S) (Value : sig
+        type t
+    end) : sig
+  (* HACK: This exposes useful functions for debugging *)
```

Comment:
can you just `: Mock_intf` instead of `include` here since there are no extra functions